### ISSUE TITLE
Error on boot: Cannot open access to console, the root account is locked

### ISSUE THREAD

**Issue by remcovg89** [piwebcam/PiWebcam#53]
When I boot the device I get the error (right after "started Update UTMP about Runlevel Changes"):

```
You are in rescue mode. After logging in, type "journalctl -xb" to view system logs, "systemctl reboot" to reboot, "systemctl default" or ^D to boot into default mode.

Cannot open access to console, the root account is locked.
See sulogin(8) man page for more details.

Press Enter to continue.
```

And if I press enter, everything boots just fine, but obviously I don't want a keyboard/screen attached all the time to boot it. What can be done about this?

**Comment by remcovg89** [comment]
After doing some more research I figured out where I went wrong and was able to resolve my issue.

I'm posting my mistake and solution to help others:
- On the initial boot my camera module was connected to the display module slot
- I realised my mistake after the first boot, so a display module was initialised
- I then corrected that mistake and continued as if that never happend

I resolved this issue by making a backup of my 'PiWebcam.conf' and simply reapply the piwebcam image to my SD card, then put back my 'PiWebcam.conf', and voilla. 

I've tested a few boots and they all seem to go well now.
